### DISCUSSION TITLE
How to use 'azureIdentities' parameters within Helm

### DISCUSSION THREAD

**Discussion by Borrelworst** [Azure/aad-pod-identity#1126] [Q&A] [answered]
The documentation for Helm states the following:

_azureIdentities | Map of azure identities and azure identity bindings resources to create. The map key is the AzureIdentity name._ 

I am just curious in how to use this. Does someone has an example where it created one or multiple identities and identitybindings within the helm values deployment itself ?

**Comment by Borrelworst** [comment, marked as answer]
Figured it out already, it just needs to be a yaml dict. However, to see al configurable options you should checkout this chart file:

https://github.com/Azure/aad-pod-identity/blob/master/charts/aad-pod-identity/templates/identities.yaml

**Reply by chewong** [reply]
You can check out an example at https://github.com/Azure/aad-pod-identity/blob/master/charts/aad-pod-identity/values.yaml#L258-L283.
